.github/workflows/build_docs.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: ["3.7", "3.8", "3.9"]
+        python_version: ["3.8", "3.9"]
```

Comment:
Should we build docs with `3.10` as well since we run all other tests with `3.10`?